FirstProject/new-evaluation.html

```diff
@@ -0,0 +1,536 @@
+<!doctype html>
+<html class="no-js" lang="">
+<head>
+    <link rel="stylesheet" href="css/main.css">
+    <link rel="stylesheet" href="img/">
+    <!-- Latest compiled and minified CSS -->
+    <!--<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css">-->
+
+    <!--&lt;!&ndash; jQuery library &ndash;&gt;-->
+    <!--<script src="https://ajax.googleapis.com/ajax/libs/jquery/3.2.1/jquery.min.js"></script>-->
+
+    <!-- Latest compiled JavaScript -->
+    <!--<script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js"></script>-->
+</head>
+
+<body>
+    <nav class="row">
+        <div class="col-md-12" style="border-bottom: 2px solid darkgrey; padding-top: 12px;">
+            <div class="col-md-4" style="text-align:center;">
+                <img style="float:left; width: 12%;" href="#" style="width: 40%;" src="img/logo-v2.png"/>
+            </div>
+            <div class="col-md-4" style="padding-top: 8px;">
+                <a href="#">Evaluations</a>
+                <a href="#" style="margin-left: 100px;">New Evaluation</a>
+            </div>
+            <div class="col-md-4" style="padding-top: 4px;">
+                <a href="#" style="float:right;">Logout</a>
+            </div>
+        </div>
+    </nav>
+    <div class="row">
+        <div class="container" style="margin-top: 100px;">
+            <div class="col-md-3"><input style="margin-top: 30px;  border-right: none; border-left: none; border-top: none;"type="password" name="lastname" placeholder="Candidate"/></div>
+            <div class="col-md-3"> <input style="margin-top: 30px;  border-right: none; border-left: none; border-top: none;"type="password" name="lastname" placeholder="Interviewer"/></div>
+            <div class="col-md-3"> <input style="margin-top: 30px;  border-right: none; border-left: none; border-top: none; border-bottom: none;"type="password" name="lastname" placeholder="4/22/2012"/> </div>
```

Comment:
How would you feel about only putting in the format? Another option would be to populate the `placeholder` with the formatted representation of a particular date (perhaps the current one?), but it's still early in the training for that. We'll see what we can do about it when we get to the good stuff, i.e. JavaScript. ;)